<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppInterface</class>
 <widget class="QMainWindow" name="AppInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>3</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>4</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>650</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>800</width>
     <height>550</height>
    </size>
   </property>
   <widget class="QWidget" name="ChatZone" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>551</width>
      <height>791</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>800</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/qwd83nc4xxf41.jpg);</string>
    </property>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>505</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">
  image: url(:/send-message.png);  
background:transparent;
 background-repeat: no; 
  width: 200px;
  height: 50px;
  padding: 0;
  border: none;
  cursor: pointer; 
 display: inline-block;


</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextLineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>507</y>
       <width>371</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
font: italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(255,255,255);

 background-repeat: no; 
  width: 200px;
  height: 50px;
  padding: 0;
  border: none;
  cursor: pointer; 
 display: inline-block;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AttachButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>500</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/attach.png); 
background:transparent;
 background-repeat: no; 
  width: 200px;
  height: 50px;
  padding: 0;
  border: none;
  cursor: pointer; 
 display: inline-block;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>271</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
    color: #1E90FF;
    font-family: Arial;
    font-size: 24px;
    font-weight: bold;
    text-align: center;
    border: none;</string>
    </property>
    <property name="text">
     <string>       Salut!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
font: italic 9pt &quot;Segoe UI&quot;;
background-color: rgb(245,245,245);
color:	#1E90FF; 
</string>
    </property>
   </widget>
   <widget class="QWidget" name="ScrollZone" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>271</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; </string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>89</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/ChatG.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddFriendButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/add-user.png); 
border: none; 
cursor: pointer; 
 background-repeat: no;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateGroupButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/groups.png);
border: none; 
cursor: pointer; 
 background-repeat: no;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
